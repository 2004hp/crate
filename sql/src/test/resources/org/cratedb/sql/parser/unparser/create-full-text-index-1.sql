CREATE INDEX text1 ON t1 using FULLTEXT(c1, "c2", t1.c3, "t1"."c4")