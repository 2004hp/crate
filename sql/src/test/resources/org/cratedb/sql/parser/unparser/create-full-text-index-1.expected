CREATE INDEX text1 ON t1 USING FULLTEXT(c1, c2, t1.c3, t1.c4)